api/test/robot_tests/web_on_chain/web_on_chain.robot

```diff
@@ -46,14 +46,33 @@ POST /websiteUploader/upload
 
     Should Be Equal As Strings    ${response.json()['address']}    ${WEBSITE_ADDRESS}
 
-GET /all/domains 
-    ${response}=    GET
-    ...    ${API_URL}/all/domains
-    ...    expected_status=${STATUS_OK}
-    Should Contain    ${response.text}    flappy
-
 GET /my/domains/{nickname}
     ${response}=    GET
     ...    ${API_URL}/my/domains/${WALLET_NICKNAME}
     ...    expected_status=${STATUS_OK}
     Should Contain    ${response.text}    ${website_url}
+
+Fetch Current DNS
+    ${response}=    GET    
+    ...    ${API_URL}/massa/node
```

Comment:
I can't find robot tests of this endpoints, could you open an issue to do it ?

Reply:
Same for  `GET /network` and `POST /network/{network}`

Reply:
https://github.com/massalabs/station/issues/1054